docs/carbon-aware-cli.md

```diff
@@ -96,4 +98,4 @@ output:
 // ...
 {"Location":"eastus","Time":"2022-07-31T12:45:11+00:00","Rating":73,"Duration":"08:00:00"},
 {"Location":"eastus","Time":"2022-07-31T20:45:11+00:00","Rating":84,"Duration":"08:00:00"}]
-```
+```
```

Comment:
Let's add more examples here for how to use `best` and `average`. I think we can grab them from the design proposal doc without much alteration.